backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/model/ProductsProposalRow.java

```diff
@@ -162,6 +172,8 @@ private ProductsProposalRow(
 		this.packingDescription = packingDescription;
 		this.packingMaterialId = packingMaterialId;
 		this.asiDescription = asiDescription != null ? asiDescription : ProductASIDescription.NONE;
+		this.asiId = asiId;
```

Comment:
Do you still need this here?

Reply:
I do, I use it here: de.metas.ui.web.order.products_proposal.service.OrderLinesFromProductProposalsProducer#createAndInitHUPackingAware

Reply:
https://github.com/metasfresh/metasfresh/pull/16573/files#diff-2370797d2c7fc150160828601f4b71229a931ec0a29319910b66cf8d12c6f692R190